test/unit/options.spec.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+describe('options', function () {
+  describe('#defaultOptionsPath', function () {
+    var options = require('../../bin/options');
+
+    it("property should have value 'test/mocha.opts'", function () {
+      expect(options.defaultOptionsPath).to.equal('test/mocha.opts');
+    });
+
+    it('property should NOT be writable/configurable', function () {
+      expect(function () {
+        options.defaultOptionsPath = '/new/path/mocha.opts';
+      }).to.throwError(function (error) {
+        expect(error).to.be.a(TypeError);
+      });
+    });
+  });
+
+  describe('#path', function () {
+    var options = require('../../bin/options');
+
+    it('should return user passed options path if it exists, or the default path', function () {
+      var sinon = require('sinon');
+      var argv, path;
+
+      argv = [];
+      sinon.spy(argv, 'indexOf');
+      path = options.path(argv, options.defaultOptionsPath);
+      expect(argv.indexOf.calledOnce).to.be.true;
+      expect(path).to.equal(options.defaultOptionsPath);
+
+      argv = ['--opts', '/user/passed/mocha.opts'];
+      sinon.spy(argv, 'indexOf');
+      path = options.path(argv, options.defaultOptionsPath);
```

Comment:
I don't really see the value in testing this

Reply:
It doesn't hurt though? :-)

Reply:
I have dropped `sinon` and assertions that depend on it.